package.json

```diff
@@ -7,21 +7,28 @@
   "scripts": {
     "compile": "tsc -p tsconfig.json",
     "lint": "tslint -p tslint.json",
-    "test": "yarn run lint",
+    "test": "nyc mocha",
+    "prepare": "tsc",
+    "pretest": "tsc",
     "prepublish": "yarn run compile && yarn run lint",
-    "package": "yarn run compile && pkg ."
+    "package": "yarn run compile && pkg .",
+    "watch": "mocha-typescript-watch"
   },
   "bin": {
-    "metricly": "js/bin/metricly.js"
+    "metricly": "js/ts/bin/metricly.js"
   },
   "author": "TheConnMan",
   "license": "ISC",
   "dependencies": {
+    "@types/chai": "^4.1.2",
```

Comment:
All `@types/`, mocha, chai, and nyc should be dev dependencies, not dependencies.